tests/models/downsample/test_down_sample_meta_data.py

```diff
@@ -46,4 +46,4 @@ def test_downsample_meta_data_pass_checks(
     # THEN all necessary models to run the down sample command are created
     assert meta_data.downsampled_sample.internal_id == f"{sample.internal_id}DS{number_of_reads}M"
```

Comment:
the command currently adds the sample with a suffix like DS5C0M, why the C? is that for "comma", mixing english (DS) and swedish (C)?

Reply:
@karlnyr changed it to this. I will not address it in this PR.